py-polars/tests/unit/io/cloud/test_aws.py

```diff
@@ -73,7 +74,6 @@ def test_read_s3(s3: str, function: Callable[..., Any], extension: str) -> None:
     ("function", "extension"),
     [
         (pl.scan_ipc, "ipc"),
-        (pl.scan_parquet, "parquet"),
```

Comment:
We cannot make it work with dummy expression because object store seems to always go to aws. This hard-coded URL seems related: https://github.com/apache/arrow-rs/blob/407e575f41365b73a84fb2a2150f918e6dab2bbe/object_store/src/aws/mod.rs#L93